api/tests/unit/domain/usecases/add-pix-authentication-method-by-email_test.js

```diff
@@ -62,5 +64,39 @@ describe('Unit | UseCase | add-pix-authentication-method-by-email', function ()
       // then
       expect(encryptionService.hashPassword).to.have.been.calledWith(generatedPassword);
     });
+
+    it('should add Pix authentication method', async function () {
```

Comment:
**suggestion**
Ce commit et le précédent peuvent être fusionné je pense. L'intension du commit précédent était un peu floue, on a généré un mot de passe mais en soit on ne s'en servait pas puisque le but final est de l'enregistrer dans la BDD (ce que fait ce commit). 
De même, je propose de faire des deux `it` qu'un seul :)